### PR TITLE
fix(console): reduce initial load time

### PR DESCRIPTION
**Pull request by peintnermax**
This reduces the initial payload of the `listMyProjectOrgs` to get the active org by setting the limit of the initial orgs to 100.

Partial of #8272

### CODE REVIEW

File: console/src/app/modules/search-user-autocomplete/search-user-autocomplete.component.ts [hunk 1]
```diff
@@ -26,6 +26,8 @@ export enum UserTarget {
   EXTERNAL = 'external',
 }
 
+const USER_LIMIT = 100;
```
Comment (eliobischof) [inline console/src/app/modules/search-user-autocomplete/search-user-autocomplete.component.ts:29]: Why are 10 not enough?
Are 100 users cached?

File: console/src/app/modules/user-grants/user-grants.component.ts [hunk 2]
```diff
@@ -306,17 +306,15 @@ export class UserGrantsComponent implements OnInit, AfterViewInit {
...
-    // Check if user has permissions for that org before changing active org
-    if (this.myOrgs.find((org) => org.id === grant.grantedOrgId)) {
-      this.authService.setActiveOrg(org);
+  public async showUser(grant: UserGrant.AsObject) {
+    const orgQuery = new OrgQuery();
+    const orgIdQuery = new OrgIDQuery();
+    orgIdQuery.setId(grant.grantedOrgId);
+    orgQuery.setIdQuery(orgIdQuery);
+
+    const orgs = (await this.authService.listMyProjectOrgs(1, 0, [orgQuery])).resultList;
```
Comment (eliobischof) [inline console/src/app/modules/user-grants/user-grants.component.ts:315]: Should this.myOrgs be updated?

File: console/src/app/services/grpc-auth.service.ts [hunk 3]
```diff
@@ -249,36 +251,49 @@ export class GrpcAuthService {
...
-        const toFind = orgs.find((tmp) => tmp.id === id);
-        if (toFind) {
-          this.setActiveOrg(toFind);
-          return Promise.resolve(toFind);
+        const orgQuery = new OrgQuery();
+        const orgIdQuery = new OrgIDQuery();
+        orgIdQuery.setId(id);
+        orgQuery.setIdQuery(orgIdQuery);
+
+        const orgs = (await this.listMyProjectOrgs(ORG_LIMIT, 0, [orgQuery])).resultList;
```
Comment (eliobischof) [inline console/src/app/services/grpc-auth.service.ts:259]: update this.cachedOrgs?

File: console/src/app/modules/search-user-autocomplete/search-user-autocomplete.component.ts (hunk 1 above)
Comment (livio-a) [reply console/src/app/modules/search-user-autocomplete/search-user-autocomplete.component.ts:29]: 100 users for a selection does not make sense to me either

File: console/src/app/services/grpc-auth.service.ts (hunk 3 above)
Comment (livio-a) [reply console/src/app/services/grpc-auth.service.ts:259]: same as with the other specific org quest i guess

File: console/src/app/modules/search-user-autocomplete/search-user-autocomplete.component.ts (hunk 1 above)
Comment (peintnermax) [reply console/src/app/modules/search-user-autocomplete/search-user-autocomplete.component.ts:29]: lets do 25 😉

**Review by eliobischof — APPROVED** [review]

**Pull request merged**